### ISSUE TITLE
Does not work with PyTest

### ISSUE THREAD

**Issue by wvxvw** [p1c2u/openapi-spec-validator#65]
See this ticket in PyTest bug tracker: https://github.com/pytest-dev/pytest/issues/4882 .

I don't know how you built the distributable package, but it looks like this:

```
Metadata-Version: 2.0
Name: openapi-spec-validator
Version: 0.2.6
Summary: UNKNOWN
Home-page: https://github.com/p1c2u/openapi-spec-validator
Author: Artur Maciag
Author-email: maciag.artur@gmail.com
License: Apache License, Version 2.0
Platform: UNKNOWN
Classifier: Development Status :: 4 - Beta
Classifier: Intended Audience :: Developers
Classifier: Topic :: Software Development :: Libraries :: Python Modules
Classifier: Operating System :: OS Independent
Classifier: Programming Language :: Python :: 2.7
Classifier: Programming Language :: Python :: 3.4
Classifier: Programming Language :: Python :: 3.5
Classifier: Programming Language :: Python :: 3.6
Classifier: Programming Language :: Python :: 3.7
Requires-Dist: jsonschema (<3)
Requires-Dist: PyYAML (>=3.13)
Requires-Dist: six
Requires-Dist: pathlib

# OpenAPI Spec validator
...
```

So, it installs `pathlib` regardless of what version of Python is used (interestingly, it works differently with different versions of Python / setuptools), but Python 7.1 `setuptools` 40.6.2 don't work with your library (downloaded from pypi).

**Comment by RonnyPfannschmidt** [comment]
as per https://pypi.org/project/pathlib/ pathlib is a unmaintained backport and pathlib2 should be used

**Comment by wvxvw** [comment]
Well, don't tell me this... I didn't use it :) The problem is *it is used by others*, and you cannot avoid it. (I'm kind of surprised I'm the first one to run into this problem, it's like nobody used Swagger these days anymore?).

**Comment by p1c2u** [comment]
It was resolved with #68 Next release should fix it. Cheers.

**Comment by p1c2u** [comment]
Fix released with version 0.2.7`. Closing.
